### DISCUSSION TITLE
Spaces and tabs in variables

### DISCUSSION THREAD

**Discussion by oscarmoutinho** [emailjs-com/help#166] [Q&A] [answered]
I all,

I'm starting with EmailJS.
I was expecting that the variables content sent in parameters were not altered.
When I use a sequence of spaces or tabs, EmailJS just 'uses' one space (similar to normal html).
Example: "aaa        aaa" -> "aaa aaa"
Is ths the expected behaviour?
If yes, how can I use several spaces or tabs in sequence ?

Thanks,
Oscar

**Comment by oscarmoutinho** [comment]
Even here it happens.
I wrote 5 spaces.
The example [(space) should be " "]: "aaa(space)(space)(space)(space)(space)aaa" -> "aaa aaa"

**Reply by xr0master** [reply, marked as answer]
This is the correct HTML code. For indents, you may check [the CSS margin](https://developer.mozilla.org/en-US/docs/Web/CSS/margin)
